src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml

```diff
@@ -20,6 +20,10 @@
                 <file name="com-github-philippefichet-sonarlint4netbeans-SonarLintParserResultTaskFactory.instance">
                 </file>
             </folder>
+            <folder name="x-html">
```

Comment:
From https://developer.mozilla.org/fr/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types, text/x-html does not exist and not work with netbeans annotation on a .html file.
On Ubuntu, mimetype for html are:
```
find /usr/share/mime -name "*html*"
/usr/share/mime/application/xhtml+xml.xml
/usr/share/mime/application/vnd.ms-htmlhelp.xml
/usr/share/mime/text/html.xml
/usr/share/mime/text/htmlh.xml
```
Wouldn't it be more just "html"?

Reply:
Hi Philippe

maybe. I have tested the update on Windows 10 and I have not seen any difference.
The correct mime is HTML, so I have fixed the XML

Can you check if on Ubuntu now works fine?

Excuse me for the mistake

all the best
matteo baccan

Reply:
The typemime is used here to launch an analysis and display the results as annotations (icon on the left in the editor).
It works perfectly under Ubuntu ;)
Thanks for your pull request :) 👍

Reply:
Very good :)

I have also try to add some other analyzers without success, I'll retry in future

ciao
matteo

Reply:
Indeed, it is a limitation of this class https://github.com/SonarSource/sonarlint-core/blob/master/core/src/main/java/org/sonarsource/sonarlint/core/client/api/common/Language.java
All plugins can't be added :(